README.markdown

```diff
@@ -7583,7 +7710,7 @@ coroutine.create
 ----------------
 **syntax:** *co = coroutine.create(f)*
 
-**context:** *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, init_by_lua&#42;, ngx.timer.&#42;, header_filter_by_lua&#42;, body_filter_by_lua&#42;, ssl_certificate_by_lua&#42;*
+**context:** *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, init_by_lua&#42;, ngx.timer.&#42;, header_filter_by_lua&#42;, body_filter_by_lua&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;, ssl_session_store_by_lua&#42;*
```

Comment:
@agentzh `coroutine.create` can not be used in `ssl_session_fetch_by_lua` and `ssl_session_store_by_lua` now.
Because it will failed in `ngx_http_lua_check_context`: https://github.com/openresty/lua-nginx-module/blob/master/src/ngx_http_lua_coroutine.c#L76
I think we can let `coroutine.create` used in `ssl_session_fetch_by_lua` by change the context check.